### PR TITLE
add status-notes and fix uniqueness

### PR DESCRIPTION
**Pull request by cjdcordeiro**
fixes https://github.com/nuvla/api-server/issues/642

### CODE REVIEW

File: code/src/sixsq/nuvla/server/resources/nuvlabox_cluster.clj [hunk 1]
```diff
@@ -81,13 +81,16 @@ The `nuvlabox-cluster` resource represents a cluster of at least one NuvlaBox
...
-                      (utils/get-matching-nuvlaboxes (:managers current)))]
-    (utils/complete-cluster-details edit-impl nb-workers nb-managers request)))
+        cluster-managers  (or managers (:managers current))
+        cluster-workers   (or workers (:workers current))
+        nb-workers        (utils/get-matching-nuvlaboxes cluster-workers)
+        nb-managers       (utils/get-matching-nuvlaboxes cluster-managers)
+        total-nodes       (+ (count cluster-managers) (count cluster-workers))
+        total-nb-nodes    (+ (count nb-managers) (count nb-workers))
+        status-notes      (cond-> []
+                            (> total-nb-nodes total-nb-nodes) (conj "WARNING: there are more NuvlaBox instances than actual nodes")
```
Comment (0xbase12) [inline code/src/sixsq/nuvla/server/resources/nuvlabox_cluster.clj:91]: I think there is a hiccup there

Comment (cjdcordeiro) [reply code/src/sixsq/nuvla/server/resources/nuvlabox_cluster.clj:91]: what is it?

Comment (cjdcordeiro) [reply code/src/sixsq/nuvla/server/resources/nuvlabox_cluster.clj:91]: ah, wrong var name

Comment (0xbase12) [reply code/src/sixsq/nuvla/server/resources/nuvlabox_cluster.clj:91]: same variable comparaison (> total-nb-nodes total-nb-nodes)

**Pull request merged**